pkg/arvo/app/launch.hoon

```diff
@@ -60,29 +63,33 @@
 ++  on-init
   ^-  (quip card _this)
   :_  this(state *[%2 state-two])
-  [%pass / %arvo %e %connect [~ /] %launch]~
+  :~  (launch-who q.byk.bol)
+      [%pass / %arvo %e %connect [~ /] %launch]
+  ==
 ::
 ++  on-save  !>(state)
 ::
 ++  on-load
   |=  old=vase
   ^-  (quip card _this)
   =/  old-state  !<(versioned-state old)
+  =|  cards=(list card)
   |-
   ?-    -.old-state
       %0
-    $(old-state [%1 tiles.old-state data.old-state path-to-tile.old-state %.n])
+    $(old-state [%1 tiles data path-to-tile %.n]:old-state)
   ::
       %1
     =/  new-state=state-two
-      :*  (~(del in tiles.old-state) [%contact-view /primary])
-          (~(del by data.old-state) %contact-view)
-          (~(del by path-to-tile.old-state) /primary)
-          first-time.old-state
+      =>  [old-state .]
+      :*  (~(del in tiles) [%contact-view /primary])
+          (~(del by data) %contact-view)
+          (~(del by path-to-tile) /primary)
+          first-time
       ==
-    $(old-state [%2 new-state])
+    $(old-state [%2 new-state], cards [(launch-who q.byk.bol) cards])
```

Comment:
launch version `%2` is already on the livenet I believe (see: [version](https://github.com/urbit/urbit/blob/93c881cec99d79b73c0f6b3ff43dd9766423604d/pkg/arvo/app/launch.hoon) from the release tagged commit), so this isn't gonna get run for people who are up to date already. You'll probably want to add a version `%3`.